start/common/src/main/java/org/ethereum/beacon/start/common/util/StorageUtils.java

```diff
@@ -0,0 +1,46 @@
+package org.ethereum.beacon.start.common.util;
+
+import org.ethereum.beacon.chain.BeaconTuple;
+import org.ethereum.beacon.chain.storage.BeaconChainStorage;
+import org.ethereum.beacon.consensus.BeaconChainSpec;
+import org.ethereum.beacon.consensus.BeaconStateEx;
+import org.ethereum.beacon.core.BeaconBlock;
+import org.ethereum.beacon.core.state.Checkpoint;
+import tech.pegasys.artemis.ethereum.core.Hash32;
+
+/**
+ * Utility functions to initialize storage from an initial state.
+ */
+public class StorageUtils {
```

Comment:
We might want to make this util a part of `chain` module to make it self-sufficient in terms of chain initialization.

Reply:
Yes. This also will help fixing failed tests.